packages/next-playwright/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "@next/playwright",
```

Comment:
Could we keep this in the main package while it's being validated/tested?

Reply:
Done